superset-frontend/src/SqlLab/components/TemplateParamsEditor.jsx

```diff
@@ -94,54 +75,45 @@ export default class TemplateParamsEditor extends React.Component {
         </a>{' '}
         syntax.
       </p>
-    );
-  }
+      <StyledConfigEditor
+        keywords={[]}
+        mode={language}
+        minLines={25}
+        maxLines={50}
+        onChange={throttle(onChange, 200)}
```

Comment:
This is great. I tested with both throttle and debounce, and it seems that debounce actually is more seamless. We should prob use that instead. Sorry about that. :)

Reply:
No problem at all, I'm on it! :)